front_end/css/styles.css

```diff
@@ -45,6 +58,137 @@
     vertical-align: middle;
 }
 
+.student-content {
+    font-size: 14px;
+    border: 2px solid  #00b386;
+    width: 100%;
+    border-left: 12px solid  #00b386;
+    border-radius: 5px;
+    padding: 14px;
+    background-color: white;
+}
+
+.instructor-content {
+    font-size: 14px;
+    border: 2px solid  #19334d;
+    width: 100%;
+    border-left: 12px solid  #19334d;
+    border-radius: 5px;
+    padding: 14px;
+    background-color: white;
+}
+
+.extra-content {
+    font-size: 14px;
+    border: 2px solid  #0095e3;
+    width: 100%;
+    border-left: 12px solid  #0095e3;
+    border-radius: 5px;
+    padding: 14px;
+    background-color: white;
+}
+
+.comment {
+    font-size: 14px;
+    border: 2px solid  #0095e3;
+    width: 100%;
+    border-radius: 5px;
+    padding: 14px;
+    background-color: white;
+}
+
+.light-heading {
+    font-weight: normal !important;
+}
+
+.help_request_div {
+    background-color: #f5f5f5;
+    border: 2px solid  #f5f5f5;
+    border-radius: 15px;
+    margin-top: 25px;
+    padding: 20px;
+}
+
+.suggestion-div {
+    background-color: #f5f5f5;
+    border: 2px solid  #f5f5f5;
+    border-radius: 15px;
+    margin-top: 10px;
+    padding: 20px;
+}
+
+.space-buttons {
+    margin-top: 10px;
+}
+
+td > button {
+    border: none;
+    background-color: white;
+    cursor: pointer;
+}
+
 .ace_gutter {
     z-index: auto !important;
+}
+
+#filter_container {
```

Comment:
If you think we might want this code in the future, it's fine to leave it here commented. If not, please remove it. If you do keep it, please leave a comment on what it does so that we will know in the future what it does.

Reply:
This isn't commented out. When a style starts with a #, that means it's connected to an id rather than a class.